Sources/ConsoleKit/Utilities/AnySendableHashable.swift

```diff
@@ -17,3 +17,10 @@ extension AnySendableHashable: CustomStringConvertible, CustomDebugStringConvert
     public var debugDescription: String { self.wrappedValue.debugDescription }
     public var customMirror: Mirror { self.wrappedValue.customMirror }
 }
+
+extension Dictionary where Key == AnySendableHashable {
+    public subscript(key: some Hashable & Sendable) -> Value? {
+        get { self[AnySendableHashable(key)] }
+        set { self[AnySendableHashable(key)] = newValue }
```

Comment:
```suggestion
        @inlinable get { self[AnySendableHashable(key)] }
        @inlinable set { self[AnySendableHashable(key)] = newValue }
```
I'm just barely willing to not declare it `@_transparent` outright 😆

Add `@inlinable` to `AnySendableHashable`'s other initializers as well, and mark the `wrappedValue` property `@usableFromInline`. Should yield at least a minor speedup for apps with large numbers of routes.